### DISCUSSION TITLE
Lock weeks for editing

### DISCUSSION THREAD

**Discussion by okms** [Puzzlepart/did#877] [Feature request]
- [x] I have searched for existing issues to avoid creating duplicates

### TLDR
Certain roles can close weeks for further edit

### Description
There are situations where we should be able to close a week for further edit. E.g. after a week has been billed to a customer and the bill has been approved.

### Relevant issues
#62 
#593

**Comment by pzljanb** [comment]
Admin should be able to Lock all Confirmed week from being "unconfirmed"
Manager should be able to Lock all Confirmed week of Subordinates from being "unconfirmed"

Admin should be able to remove Lock of all users
